spectool/specdata.xml

```diff
@@ -88,7 +88,7 @@
   <element name="DefaultDuration" cppname="TrackDefaultDuration" level="3" id="0x23E383" type="uinteger" minver="1" range="not 0">Number of nanoseconds (not scaled via TimecodeScale) per frame ('frame' in the Matroska sense -- one element put into a (Simple)Block).</element>
   <element name="DefaultDecodedFieldDuration" cppname="TrackDefaultDecodedFieldDuration" level="3" id="0x234E7A" type="uinteger" minver="4" range="not 0">The period in nanoseconds (not scaled by TimcodeScale)
 between two successive fields at the output of the decoding process (see <a href="http://www.matroska.org/technical/specs/notes.html#DefaultDecodedFieldDuration">the notes</a>)</element>
-  <element name="TrackTimecodeScale" level="3" id="0x23314F" type="float" mandatory="1" minver="1" maxver="3" webm="0" default="1.0" range="&gt; 0">DEPRECATED, DO NOT USE. The scale to apply on this track to work at normal speed in relation with other tracks (mostly used to adjust video speed when the audio length differs).</element>
+  <element name="TrackTimecodeScale" level="3" id="0x23314F" type="float" mandatory="1" minver="1" maxver="3" webm="0" default="1.0" range="&gt; 0x0p+0">DEPRECATED, DO NOT USE. The scale to apply on this track to work at normal speed in relation with other tracks (mostly used to adjust video speed when the audio length differs).</element>
```

Comment:
updated and squashed in https://github.com/Matroska-Org/foundation-source/pull/16/commits/28a63aa9c1fc7efa2e09950bb0a5c9593d7c5b21